doc/develop.rst

```diff
@@ -61,6 +61,7 @@ Configuration Tutorials
    user-guides/kernel-parameters
    user-guides/acrn-dm-parameters
    misc/packaging/README
+   tutorials/tarball_installation
```

Comment:
since we changed the title not to say tarball, how about changing the file name too